platform/src/main/java/org/stellar/anchor/platform/NettyCustomerIntegration.java

```diff
@@ -1,118 +1,172 @@
 package org.stellar.anchor.platform;
 
 import com.google.gson.Gson;
-import io.netty.handler.codec.http.QueryStringEncoder;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import lombok.SneakyThrows;
+import okhttp3.*;
+import org.stellar.anchor.exception.*;
 import org.stellar.anchor.integration.customer.CustomerIntegration;
 import org.stellar.anchor.platform.model.Customer;
 import org.stellar.anchor.platform.model.CustomerStatus;
 import org.stellar.anchor.platform.repository.CustomerRepository;
-import org.stellar.platform.apis.callbacks.responses.DeleteCustomerResponse;
+import org.stellar.platform.apis.callbacks.requests.PutCustomerRequest;
 import org.stellar.platform.apis.callbacks.responses.GetCustomerResponse;
 import org.stellar.platform.apis.callbacks.responses.PutCustomerResponse;
-import reactor.core.publisher.Mono;
-import reactor.netty.ByteBufMono;
-import reactor.netty.http.client.HttpClient;
+import org.stellar.platform.apis.shared.ErrorResponse;
 
 public class NettyCustomerIntegration implements CustomerIntegration {
-  private final String baseUri;
+  private final HttpUrl customerUrl;
   private final Gson gson = new Gson();
   private final CustomerRepository customerRepository;
+  private final OkHttpClient httpClient;
 
   public NettyCustomerIntegration(String baseUri, CustomerRepository customerRepository) {
     try {
       new URI(baseUri);
     } catch (URISyntaxException e) {
       throw new IllegalArgumentException("invalid 'baseUri'");
     }
-    this.baseUri = baseUri;
+    HttpUrl baseUrl = HttpUrl.parse(baseUri);
```

Comment:
You are using okhttp here. Since we are using Netty, I suggest that you use the Netty way.